.github/workflows/toffee.yaml

```diff
@@ -0,0 +1,40 @@
+name: push default page to Toffee static webapp
+run-name: ${{ github.ref_name }} ${{ github.actor }}
+
+on:
+  workflow_dispatch:
+  check_run:
+    types:
+      - completed
+  push:
+    branches:
+      - master
+    paths:
+      - 'default/**'
+      - .github/workflows/toffee.yaml
+
+  pull_request:
+    types: [opened]
+    branches:
+      - master
+    paths:
+      - 'default/**'
+      - .github/workflows/toffee.yaml
+
+jobs:
+  build_and_deploy_job:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    name: Build and Deploy Job
+    steps:
```

Comment:
are you planning to add a re-usable workflow in this PR or a follow up?

we don't want to maintain this workflow 30 times instead we want a workflow like this:
https://github.com/jenkinsci/slack-plugin/blob/master/.github/workflows/cd.yaml